### PR TITLE
Disable nagle on whois sockets

### PR DESCRIPTION
**Pull request by troy2914**
This incorporates https://github.com/haussli/irrd4 with a fixed
junit test.

### CODE REVIEW

**Comment by troy2914** [comment]
waiting on Could not submit coverage: 422 Client Error: Unprocessable Entity for url: https://coveralls.io/api/v1/jobs

File: irrd/server/whois/protocol.py [hunk 1]
```diff
@@ -35,6 +35,9 @@ def connectionMade(self) -> None:  # noqa: N802
             self.transport.loseConnection()
             return
 
+        # disable nagle
```
Comment (mxsasha) [inline irrd/server/whois/protocol.py]: I think we can remove this comment line, as it merely describes precisely what is happening in the next line.
(I don't seem to have permission to edit your PR myself.)

Comment (troy2914) [reply irrd/server/whois/protocol.py]: The line mentions TcpNoDelay,  which just so happens setting that also disables nagle.

This was a line from the original author, I actually considered changing it to "disables nagle".
I'm flexible if you think it's still unneeded.

Comment (mxsasha) [reply irrd/server/whois/protocol.py]: OK, let's keep it as is. It's actually not important anyways, the socket handling is quite different in 4.1, so this entire file won't exist in 4.1 anymore.

**Pull request merged**